backend/routes/superadmin_routes.py

```
@@ -0,0 +1,828 @@
+
+        return jsonify({
+            "structures": [structure.to_dict() for structure in structures],
+            "pagination": {
+                "page": page,
+                "per_page": per_page,
+                "total": total,
+                "pages": (total + per_page - 1) // per_page
+            }
+        }), 200
+
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)